utilities/ldap_sync.sql

```diff
@@ -1,11 +1,138 @@
-/* 
-The below scripts will help you synchronize database users with LDAP users. The script will search for roles that contain 
+/*
+======================================
+---------      Overview     ----------
+======================================
+The below scripts will help you synchronize database users with LDAP users. This solution will search for roles that contain 
 a distinguished name as a role comment and will then pull the members of this group from LDAP, create the necessary users, 
 and grant permissions. When users are removed from the group in LDAP, the users will also have the role revoked from them
 
 For more information, please see the below community article:
 https://community.exasol.com/t5/database-features/synchronization-of-ldap-active-directory-groups-and-members-to/ta-p/1686
 
+======================================
+--------- Where to begin?   ----------
+======================================
+About:  These are suggestions to ease implementation of the LDAP Sync process.
+
+1) At the top of each script/udf is "open schema xxxxx". Set the schema in this script,
+   GET_AD_ATTRIBUTES and LDAP_HELPER. The default is to create and use the schema "EXA_TOOLBOX".
+   Your use case will determine the appropriate schema. This is important, as we have 
+   remove the hardcoded schemas in each of the scripts provided, as they previously were using 
+   the default schema "EXA_TOOLBOX".
+   
+2) Information to compile:
+   a) DSN Name and I.P. address of the LDAP Server, including the port. Upon initial 
+      implementation, we suggest using the unsecured LDAP port. Later, you can make
+      changes to implement the secured LDAP configurations.
+   b) An LDAP user, and password, which has permissions to query the LDAP Server.
+   c) A valid LDAP Distinguished Name record, for either a user or a group. This will be used 
+     when confirming the LDAP connection and running "LDAP_HELPER" with
+     the LDAP query criteria. Example for LDAP user query which "LDAP_HELPER"
+     can use:
+         'cn=John Doe,ou=Users,dc=example,dc=com', 'uid'
+
+3) Ensure Exasol's EXAOperation UI has the DNS servers filled in, along with the "LDAP Server URLs"
```

Comment:
As EXAOperation is to cease relatively soon (at least for new installations, which mostly will be reading this), how about replacing it with something more generic like "EXAOperation UI and/or via confd_client"? Appears multiple times.